chromium/popup.js

```diff
@@ -65,11 +72,17 @@ function appendRuleLineToListDiv(ruleset, list_div) {
       break;
     }
   }
-  var xhr = new XMLHttpRequest();
-  try {
-    xhr.open("GET", favicon.src, true);
-    label.appendChild(favicon);
-  } catch (e) {}
+  if (favicon.src == "chrome://favicon/") {
+    favicon.src += "https://" + hostnames[ruleset.name];
+  }
+
+  if (navigator.userAgent.match("Chrome")) {
+    var xhr = new XMLHttpRequest();
+    try {
+      xhr.open("GET", favicon.src, true);
```

Comment:
Remove this XMLHTTPRequest stuff. I'm not sure what it was supposed to do at all.